projects/control/eagerx/install_eagerx.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+if [[ -z "$OPENDR_HOME" ]]; then
+       echo "OPENDR_HOME is not defined"
+       exit 1
+fi
+
+if [[ -z "$ROS_DISTRO" ]]; then
+       echo "ROS_DISTRO is not defined"
+       exit 1
+fi
+
+MODULE_PATH=${OPENDR_HOME}/projects/control/eagerx
+WS_PATH=${OPENDR_HOME}/lib/catkin_ws_eagerx
+
+## ROS
+sudo apt-get update && sudo apt-get install -y \
+  ros-${ROS_DISTRO}-ros-base \
+  python3-rosdep || exit;
+source /opt/ros/${ROS_DISTRO}/setup.bash
+
+# Check if EAGERx submodule is initialized
+if [ ! -d ${MODULE_PATH}/eagerx/eagerx_core ]; then
+  STR=$'EAGERx submodule not initialized\nPlease run:\ngit submodule init\ngit submodule update'
+  echo "$STR"
+  exit 1
```

Comment:
It's good to check but this shouldn't happen. In the installation guide we'll need to mention to clone the repo with `git clone --recurse-submodules -j8 https://github.com/...`

If you intend to add a test for this (I don't think so, but if you do), for the `test tools` to work in the CI you'll need to add:

```
- uses: actions/checkout@v2
    with:
       submodules: true
```

to the `tests_sources.yml`, or `submodules: recursive` if it depends on additional submodules.

Reply:
Thank you for the clarification!